dwave/system/samplers/clique.py

```diff
@@ -274,6 +274,10 @@ def sample(self, bqm, chain_strength=None, **kwargs):
                 sampler, because it scales the problem as part of the embedding
                 process.
 
+        Returns:
+            :class:`~dimod.SampleSet` constructed from a (non-blocking) :class:`~concurrent.futures.Future`-like object: A
+            :obj:`~dimod.SampleSet` object.
```

Comment:
I don't see a change here.

Reply:
It's there: 
![image](https://user-images.githubusercontent.com/34041130/123471779-2a41a080-d5ab-11eb-9b27-1cb034214509.png)

Reply:
Compare with https://github.com/dwavesystems/dwave-system/pull/408#discussion_r658944972

I was suggesting we use a consistent style for return:
```
Returns:
    type:
        Description.
```

Reply:
Ah, I have the ":class:`~dimod.SampleSet` constructed from a (non-blocking) :class:`~concurrent.futures.Future`-like object" as the definition of the type and you prefer the details moved to the description of what is returned. The former seems more accurate to me.  I'll ping you.